recipes/vs2008_runtime/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: vs2008_runtime
+  version: "9.0.30729.6161"
+
+build:
+  number: 0
+  skip: true          # [not win]
+  msvc_compiler: 9.0
+
+test:
+  commands:
+    {% set libs = [
+        "msvcm90",
+        "msvcp90",
+        "msvcr90",
+        "vcomp90"
+    ] %}
+    {% for each_lib in libs %}
+    - if not exist "%PREFIX%\\{{ each_lib }}.dll" exit 1
```

Comment:
Defaults is placing the dlls in `%PREFIX%` and `%LIBRARY_BIN%`, so did the same here

Reply:
Thanks for pointing this out.